packetfabric/metadata.csv

```diff
@@ -0,0 +1,3 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+packetfabric.ifd_traffic_rate_metric,rate,60,byte,second,ifd traffic rate metrics,0,packetfabric,ifd_traffic_rate_metric,
```

Comment:
The description should spell out what ifd and ifl traffic stands for instead of the acronym. Please update the acronym